CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-web-profile-manager ChangeLog
 
+## 10.0.1 - 2021-09-xx
+
+### Changed
+- No changes to the public API. Test dependencies now use bedrock-meter-http@3
+  and bedrock-profile-http@9.
```

Comment:
There's no reason to release a new package when this happens, IMO -- the tests are not even in the package.

Reply:
Sounds good.